models/aether_lat-lon/model_mod.nml

```diff
@@ -0,0 +1,9 @@
+&model_nml
+   template_file  = 'if other than filter_input_0001.nc'
+   debug          = 0
+   variables = 'Temperature',   'QTY_TEMPERATURE',      '1000.0',  'NA',      'UPDATE',
```

Comment:
I think these should be set to sensible defaults
Do you want to clamp temperature at a minimum of 1000? If not have 'NA' for clamping. 

similarly, set:
template_file='filter_input_0001.nc'